GothicVRProject/GothicVR
# Map texture used in a canvas parallel to the menu

# To test
1. Press primary right
2. Map should appear
3. Press it again. Map should dissapear
4. Behaviour should be similar to menu


# Checklist
(Please judge for yourself which entry is valuable for your PR)

## Scene

# Testing

## Dependencies
